rbac/rbac/celery.py

```diff
@@ -63,3 +66,12 @@
 
 # Load task modules from all registered Django app configs.
 app.autodiscover_tasks()
+
+
+@worker_ready.connect
+def start_metrics_server(sender=None, **kwargs):
+    """Start the metrics server."""
+    registry = CollectorRegistry()
+    multiprocess.MultiProcessCollector(registry)
+    generate_latest(registry)
```

Comment:
how this work ? Is there any effect as return value is not used ? [ref](https://github.com/prometheus/client_python/blob/master/prometheus_client/exposition.py#L248)

Maybe `start_http_server` is enough to serve metrics ?

Reply:
Yeah good point, I think you're right.